app.js

```diff
@@ -1,246 +1,97 @@
 }
 
 CookieStore.prototype.getAvgCookieCount = function () { // Add method with a prototype
-  var avgCookiesPerhour = Math.floor(Math.random() * ((this.maxCustomers - this.minCustomers + 1) + this.minCustomers) * this.avgCookies);
-  this.hourlyCount.push(avgCookiesPerhour);
-  console.log(this.name + 'Got avg cookie count.');
+  for (var hours = 0; hours < this.businessHours.length; hours++) {
+    var avgCookiesPerhour = Math.floor(Math.random() * ((this.maxCustomers - this.minCustomers + 1) + this.minCustomers) * this.avgCookies);
+    this.hourlyCookieCount.push(avgCookiesPerhour);
+    this.totalCountForDay += avgCookiesPerhour;
+  }
 };
 
-// var hourlyCookie = [50, 100, 23, 15, 66]; //Add array to your constructor as a parameter
+var firstAndPike = new CookieStore('First and Pike Store', 23, 65, 6.3); // CookieStore is a new type of instance.
+var seaTacAirport = new CookieStore('SeaTac Airport Store', 3, 24, 1.2); // Now, two new instances of CookieStore.
+var seattleCenter = new CookieStore('Seattle Center Store', 11, 38, 3.7);
+var capitolHill = new CookieStore('Capitol Hill Store', 20, 38, 2.3);
+var alki = new CookieStore('Alki Store', 2, 16, 4.6);
 
-var pikePlace = new CookieStore('Pike Place', 23, 65, 6.3); // CookieStore is a new type of instance.
-var otherStore = new CookieStore('Other Store', 100, 200, 10); // Now, two new instances of CookieStore.
-var anotherStore = new CookieStore('Downtown', 50, 500, 1600);
+firstAndPike.getAvgCookieCount();
+seaTacAirport.getAvgCookieCount();
+seattleCenter.getAvgCookieCount();
+capitolHill.getAvgCookieCount();
+alki.getAvgCookieCount();
 
-var stores = [pikePlace, otherStore, anotherStore];
+var stores = [firstAndPike, seaTacAirport, seattleCenter, capitolHill, alki];
 
-var tableEl = document.createElement('table');  //Step 1 - Create Element
+var storeSalesDataEl = document.createElement('table');
 
-for (var i = 0; i < stores.length; i++){
-  var currentStore = store[i];
+var timeRowEl = document.createElement('tr');
+storeSalesDataEl.appendChild(timeRowEl);
 
-  var rowEl = document.createElement('tr'); //Step 3 - Create elements to loop through.
-  tableEl.appendChild(nameEl);
+var blankCell = document.createElement('td');
+timeRowEl.appendChild(blankCell);
 
-  var nameEl = document.createElement('th'); // 'th' tag to create a heading
-  nameEl.textContent = currentStore.name;
-  rowEl.appendChild(nameEl);
+for (var cell = 0; cell < firstAndPike.businessHours.length; cell++) { // Needed time from first store to display hours.
+  var timeHour = document.createElement('td');
+  timeHour.textContent = firstAndPike.businessHours[cell];
+  timeRowEl.appendChild(timeHour);
+}
 
-  var minCustEl = document.createElement('td');
-  minCustEl.textContent = currentStore.minCustomers;
-  rowEl.appendChild(minCustEl);
+for (var storeIndex = 0; storeIndex < stores.length; storeIndex++) {
+  var tableRowEl = document.createElement('tr');
```

Comment:
from the lab instructions: Each cookie stand location should have a separate render() method that creates and appends its row to the table